packages/@webex/plugin-encryption/src/encryption/encryption.types.ts

```diff
@@ -0,0 +1,5 @@
+interface IEncryption {
```

Comment:
is it ok to add this type inline in the file , if it grows big we can move it out